### PR TITLE
Add pdf related documentation and fix default template bug

### PR DESCRIPTION
**Pull request by vicancy**
Fixing https://github.com/dotnet/docfx/issues/1715

### CODE REVIEW

File: Documentation/tutorial/walkthrough/walkthrough_generate_pdf.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+Parameters are similar to `build` section, definitely it is using a different template (the builtin template is `pdf.default`), with another output destination. We also exclude TOC files as each TOC file generates a PDF file:
+```json
+...
+  "pdf": {
+    "content": [
+      {
+        "files": [
+          "api/**.yml",
+          "api/index.md",
+          "api-vb/**.yml"
```
Comment (superyyrrzz) [inline Documentation/tutorial/walkthrough/walkthrough_generate_pdf.md]: seems the folder structure doesn't align with the zip download?

Comment (vicancy) [inline Documentation/tutorial/walkthrough/walkthrough_generate_pdf.md]: remove api/index.md, others are generated by `docfx metadata`

**Comment by superyyrrzz** [comment]
:shipit:

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**